Putting our foot down for the Nth time (a.k.a. Indentation IV: electric hullabaloo) (#2855)
psf/black: docs/faq.md

:class: this-will-duplicate-information-and-it-is-still-useful-here
```

## Why spaces? I prefer tabs

PEP 8 recommends spaces over tabs, and they are used by most of the Python community.

> There are no options to configure the indentation style, and requests for any such

Reviewer: ```suggestion
_Black_ provides no options to configure the indentation style, and requests for any such
```
There are options, just not here.

> _Black_ provides no options to configure the indentation style, and requests for any such

Author: ```suggestion
_Black_ provides no options to configure the indentation style, and requests for such
```
